crypto/ocsp/ocsp_vfy.c

```diff
@@ -84,6 +84,9 @@ int OCSP_basic_verify(OCSP_BASICRESP *bs, STACK_OF(X509) *certs,
             OCSPerr(OCSP_F_OCSP_BASIC_VERIFY, ERR_R_X509_LIB);
             goto f_err;
         }
+        if ((flags & OCSP_PARTIAL_CHAIN) != 0)
+            X509_STORE_CTX_set_flags(ctx,
+                    X509_STORE_CTX_get_flags(ctx) | X509_V_FLAG_PARTIAL_CHAIN);
```

Comment:
I just realized that there is quite some overlap between `OCSP_basic_verify()` and `OCSP_request_verify()` regarding the `X509_STORE_CTX` initialization and parameterization.
I suggest extracting a static function for doing the things common to both and using the new flag there,
such that also `OCSP_request_verify()` profits from it.

Reply:
I don't know, the flags supported are different.  I would really rather not to that in this PR.  I would add the three lines to the other function if you think they are useful.

Reply:
> I don't know, the flags supported are different. I would really rather not to that in this PR. 

Yes, since the flags are different a generalization regarding the verification flags would require an extra flags parameter.

> I would add the three lines to the other function if you think they are useful.

I'm against code redundancy, and copying the new lines to the other place would even add to this.
On the other hand, I meanwhile found that extracting (as I suggested above) a static function for doing the things common to both functions would be way beyond the scope of this PR.
So I'm going to set up a separate PR for cleaning up the OCSP verification code.

Reply:
> So I'm going to set up a separate PR for cleaning up the OCSP verification code.

I look forward to that.  Hopefully this PR will get merged and approved. Keep in mind the BETA timetable, but if all you are doing is changing internals (and not API's) you have more time. :) Adding new flags/define's have traditionally been considered an API addition.

Reply:
I've just submitted that follow-up PR: #12669.
It does not touch the API, so is not urgent before the beta.